imblearn/over_sampling/_adasyn.py

```diff
@@ -50,6 +50,15 @@ class ADASYN(BaseOverSampler):
     --------
     SMOTE : Over-sample using SMOTE.
 
+    SMOTENC : Over-sample using SMOTE for continuous and categorical features.
+
+    SMOTEN : Over-sample using the SMOTE variable specifically for categorical
```

Comment:
SMOTEN : Over-sample using the SMOTE variant specifically for nominal features only.